Products/ZenHub/zenhub.py

```diff
@@ -195,7 +198,7 @@ def perspective_getService(self,
                 service.addListener(listener, options)
             return service
 
-    def perspective_reportingForWork(self, worker):
+    def perspective_reportingForWork(self, worker, pid=None):
```

Comment:
A `ZenHub` instance has an attribute, `workers`, that is a collection of references to `zenhubworkers`, but the references are remote references and do not include any of the attributes of the referenced `zenhubworker` instance.